README.md

```diff
@@ -140,23 +140,77 @@ sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/ssl/privat
 
 *Important*: Make sure that the certificate files are named exactly `ssl-certificate.key` and `ssl-certificate.crt`. If you have procured the certificate from a different source, rename them to the prescribed filenames, and ensure that they are located in the same folder as the `install.sh` script. They will get uploaded to the Chef Habitat supervisor during the install.
 
+### Airgapped Installation (Recommended prep work, if airgapped)
+
+In order to install the on-prem Chef Habitat Builder in an airgapped (no direct Internet access) environment, the following preparatory steps are helpful:
```

Comment:
May be obvious, but worth calling out that the prep steps are to be done in a separate non-airgapped environment?

Reply:
Addressed in https://github.com/habitat-sh/on-prem-builder/pull/212/commits/38454a460a0f673bf5f0cde08392dfcb3e36b1f6